*** a/src/Application.vala	2018-07-01 00:38:09.000000000 +0200
--- b/src/Application.vala	2018-09-25 12:21:45.349816064 +0200
*************** public class Dippi : Gtk.Application {
*** 27,32 ****
--- 27,34 ----
  
      protected override void activate () {
          var app_window = new MainWindow (this);
+         Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
+         Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
          app_window.show_all ();
  
          var quit_action = new SimpleAction ("quit", null);
